docs/framework/data/adonet/sql/linq/how-to-insert-rows-into-the-database.md

```diff
@@ -42,5 +42,5 @@ You insert rows into a database by adding objects to the associated [!INCLUDE[vb
 ## See Also  
  [How to: Manage Change Conflicts](../../../../../../docs/framework/data/adonet/sql/linq/how-to-manage-change-conflicts.md)   
  [DataContext Methods (O/R Designer)](http://msdn.microsoft.com/library/c149f4e5-3b61-4c33-892e-3e26d47f3eeb)   
- [How to: Assign stored procedures to perform updates, inserts, and deletes (O/R Designer)](../Topic/How%20to:%20Assign%20stored%20procedures%20to%20perform%20updates,%20inserts,%20and%20deletes%20\(O-R%20Designer\).md)   
+ [How to: Assign stored procedures to perform updates, inserts, and deletes (O/R Designer)](http://msdn.microsoft.com/library/e88224ab-ff61-4a3a-b6b8-6f3694546cac)   
```

Comment:
Same as the previous comment: this can link to docs rather than MSDN: https://docs.microsoft.com/en-us/visualstudio/data-tools/how-to-assign-stored-procedures-to-perform-updates-inserts-and-deletes-o-r-designer